test: fix typo preventing test from running (#31931)
nodejs/node: test/common/README.md

failures. However, it is useful in some rare cases to disable it, for example if
the `unhandledRejection` hook is directly used by the test.

### `enoughTestCpu`

* [&lt;boolean>][]

> Indicates if there is more than 1 CPU and that the first CPU has a speed of at

Reviewer: ```suggestion
Indicates if there is more than 1 CPU or that the single CPU has a speed of at
```

If I read the code correctly?
